src/lib/circuit_breaker/circuit_breaker_params.c

```diff
@@ -113,7 +113,7 @@ PARAM_DEFINE_INT32(CBRK_AIRSPD_CHK, 0);
  * @category Developer
  * @group Circuit Breaker
  */
-PARAM_DEFINE_INT32(CBRK_FLIGHTTERM, 121212);
```

Comment:
Just for testing?

Reply:
Nope. the termination should always be available by default. At the moment, if someone sets "Terminate" in the drop-down menu of the "Safety" tab of QGC, flight termination will not be triggered. Flight termination was disabled by default "_for now_", 4 years ago, because "it wasn't safe" but the objective of this PR is also to review and test the flight termination logic.

Reply:
That sounds fine as long as we're sure of the implications system wide and document it clearly. Especially within the px4io firmware.

Reply:
This circuit breaker is now again enabled by default.